client/src/pages/CreateSurvey/Questions/Question.jsx

```diff
@@ -40,12 +41,49 @@ const Question = ({ questionIndex }) => {
     dispatch({ type: 'DELETE_QUESTION', payload });
   };
 
+  const handleUpClick = () => {
+    const payload = {};
+    payload.index = questionIndex;
+    dispatch({ type: 'MOVE_QUESTION_UP', payload });
+  };
+
+  const handleDownClick = () => {
+    const payload = {};
+    payload.index = questionIndex;
+    dispatch({ type: 'MOVE_QUESTION_DOWN', payload });
+  };
+
   return (
-    <>
-      <Box my={2}>
+    <Box display='flex'>
+      <Box
+        py={4}
+        display='flex'
+        flexDirection='column'
+        justifyContent='space-between'
+      >
+        {/* Only render up button if not first item */}
```

Comment:
Do you need to write this comment every time you add an arrow element?

Reply:
it was for the logic in case it was kind of confusing